get_http_test.go

```diff
@@ -215,6 +251,48 @@ func TestHttpGetter_authNetrc(t *testing.T) {
 	}
 }
 
+var cases = []struct {
```

Comment:
Same as above - this should probably go inside of the test it is used for, since we aren't using it anywhere else other than that test.